examples/federated-kaplan-meier-fitter/examplefkm/task.py

```diff
@@ -0,0 +1,14 @@
+from lifelines.datasets import load_waltons
+
+from flwr_datasets.partitioner import NaturalIdPartitioner
+from datasets import Dataset
+
+
+def load_partition(partition_id: int):
+    X = load_waltons()
```

Comment:
would it make sense to have this outside this function acting like the _caching_ we have implemented for the templates?

Reply:
I've moved the X outside of the function now; I guess that would be the best solution.